calendars/networkx.yaml

```diff
@@ -23,10 +22,24 @@ events:
       Meeting notes: https://hackmd.io/rqs_pWMxSLmICXCpI3w-Ug
       Zoom meeting ID: 941 9287 4965
       Zoom meeting passcode: 572126
-    begin: 2024-05-29 10:00:00
-    end: 2024-05-29 11:00:00
+    begin: 2024-05-29 17:00:00 +00:00
+    end: 2024-05-29 18:00:00 +00:00
     url: https://anaconda.zoom.us/j/94192874965?pwd=K0wvcmhXem41ZlVSQ2l4TXlUaDgxdz09
     repeat:
       interval:
         days: 7
-      until: 2025-12-31 00:00:00
+      until: 2025-12-31 00:00:00 +00:00
+
+  - summary: nx-parallel call
```

Comment:
I would suggest spelling out NetworkX here, too, as this will be the title of the calendar entries and nx is a bit too jargony